server/src/main/java/org/apache/druid/server/coordinator/config/CoordinatorKillConfigs.java

```diff
@@ -70,6 +71,9 @@ public class CoordinatorKillConfigs
   @JsonProperty("maxSegments")
   private final Integer killUnusedMaxSegments;
 
+  @JsonProperty("maxInterval")
+  private final Period killUnusedMaxInterval;
```

Comment:
Have you chosen `Period` instead of `Duration` since it would be more intuitive to align with segment granularities?